security/rbac/scripts/run.sh

```diff
@@ -44,4 +44,4 @@ sleep 1
 ./enable-rbac-ksqldb-server.sh
 ./enable-rbac-control-center.sh
 
-./cleanup.sh
+# ./cleanup.sh
```

Comment:
Looking back at history, `run.sh` was always intended to clean up after itself, but I don't see why it couldn't leave services running. That would also work better with the [docs page](https://docs.confluent.io/platform/current/tutorials/examples/security/rbac/docs/index.html#run-example-on-local-install-of-cp) that goes with the example since the step after `Run example` is `Stop example`

just delete the line rather than comment out though

Reply:
cleanup should be done separately for actually "running" and playing with the examples.

Deleted the line as suggested. Commented initially because it was like that in previous history.